pkg/e2e/Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM linuxkit/alpine:07f7d136e427dc68154cd5edbb2b9576f9ac5213 AS build
+
+ENV kubernetes_version v1.9.0
+
+RUN apk add -U --no-cache \
+  bash \
+  coreutils \
+  curl \
+  findutils \
+  git \
+  go \
+  grep \
+  libc-dev \
+  linux-headers \
+  make \
+  rsync \
+  && true
+
+ENV GOPATH=/go PATH=$PATH:/go/bin
+
+ENV KUBERNETES_URL https://github.com/kubernetes/kubernetes.git
+#ENV KUBERNETES_BRANCH pull/NNN/head
+ENV KUBERNETES_COMMIT ${kubernetes_version}
+RUN mkdir -p $GOPATH/src/github.com/kubernetes && \
+    cd $GOPATH/src/github.com/kubernetes && \
+    git clone $KUBERNETES_URL kubernetes
+WORKDIR $GOPATH/src/github.com/kubernetes/kubernetes
+RUN set -e; \
+    if [ -n "$KUBERNETES_BRANCH" ] ; then \
+        git fetch origin "$KUBERNETES_BRANCH"; \
+    fi; \
+    git checkout $KUBERNETES_COMMIT
+
+RUN make WHAT="cmd/kubectl test/e2e/e2e.test vendor/github.com/onsi/ginkgo/ginkgo"
+
+RUN mkdir -p /out/etc/apk && cp -r /etc/apk/* /out/etc/apk/
+#coreutils needed for du -B for disk image checks made by kubelet
+# example: $ du -s -B 1 /var/lib/kubelet/pods/...
+#          du: unrecognized option: B
```

Comment:
Is this relevant in the context of building the tests?

Reply:
No